lib/src/session.dart

```diff
@@ -71,6 +71,10 @@ class Session {
     final controller = SessionProgressNotificationsController(this, direction, mode);
     return controller.createStream();
   }
+
+  void raiseSessionError(SyncErrorCategory category, int errorCode, bool isFatal) {
```

Comment:
this should be an internal method.

Reply:
Done in https://github.com/realm/realm-dart/pull/577